pallets/author-inherent/src/lib.rs

```diff
@@ -117,7 +117,7 @@ pub mod pallet {
 		/// but before transactions are executed.
 		// This should go into on_post_inherents when it is ready https://github.com/paritytech/substrate/pull/10128
 		// TODO better weight. For now we just set a somewhat conservative fudge factor
-		#[pallet::weight((10 * T::DbWeight::get().write, DispatchClass::Mandatory))]
+		#[pallet::weight((3 * T::DbWeight::get().read + 11 * T::DbWeight::get().write, DispatchClass::Mandatory))]
```

Comment:
```suggestion
		#[pallet::weight((10 * T::DbWeight::get().write, DispatchClass::Mandatory))]
```
I'll revert the weight change so the objective optimization can get in.